File: workspace-servidor/flotix2021/src/main/java/com/flotix/dto/AlquilerDTO.java
package com.flotix.dto;

/**
 * Objeto que almacena la informacion de los alquileres para su devolucion
 * 
 * @author Flor
 *
 */
public class AlquilerDTO {

	private String id;
	private String matricula;
	private VehiculoDTO vehiculo;
	private String idCliente;
	private ClienteDTO cliente;
	private String fechaInicio;
	private String fechaFin;
	private int km;
	private String tipoKm;
	private double importe;
	private String tipoImporte;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getMatricula() {
		return matricula;
	}

	public void setMatricula(String matricula) {
		this.matricula = matricula;
	}

	public String getIdCliente() {
		return idCliente;
	}

	public void setIdCliente(String idCliente) {
		this.idCliente = idCliente;
	}

	public ClienteDTO getCliente() {
		return cliente;
	}

	public void setCliente(ClienteDTO cliente) {
		this.cliente = cliente;
	}

	public VehiculoDTO getVehiculo() {
		return vehiculo;
	}

	public void setVehiculo(VehiculoDTO vehiculo) {
		this.vehiculo = vehiculo;
	}

	public String getFechaInicio() {
		return fechaInicio;
	}

	public void setFechaInicio(String fechaInicio) {
		this.fechaInicio = fechaInicio;
	}

	public String getFechaFin() {
		return fechaFin;
	}

	public void setFechaFin(String fechaFin) {
		this.fechaFin = fechaFin;
	}

	public int getKm() {
		return km;
	}

	public void setKm(int km) {
		this.km = km;
	}

	public String getTipoKm() {
		return tipoKm;
	}

	public void setTipoKm(String tipoKm) {
		this.tipoKm = tipoKm;
	}

	public double getImporte() {
		return importe;
	}

	public void setImporte(double importe) {
		this.importe = importe;
	}

	public String getTipoImporte() {
		return tipoImporte;
	}

	public void setTipoImporte(String tipoImporte) {
		this.tipoImporte = tipoImporte;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelDTO/AlquilerDTO.cs

namespace Flotix2021.ModelDTO
{
    public class AlquilerDTO
    {
        private string _id;

        public string id
        {
            get { return _id; }
            set { _id = value; }
        }

        private string _idVehiculo;

        public string idVehiculo
        {
            get { return _idVehiculo; }
            set { _idVehiculo = value; }
        }

        private VehiculoDTO _vehiculo;

        public VehiculoDTO vehiculo
        {
            get { return _vehiculo; }
            set { _vehiculo = value; }
        }

        private string _idCliente;

        public string idCliente
        {
            get { return _idCliente; }
            set { _idCliente = value; }
        }

        private ClienteDTO _cliente;

        public ClienteDTO cliente
        {
            get { return _cliente; }
            set { _cliente = value; }
        }


        private string _fechaInicio;

        public string fechaInicio
        {
            get { return _fechaInicio; }
            set { _fechaInicio = value; }
        }

        private string _fechaFin;

        public string fechaFin
        {
            get { return _fechaFin; }
            set { _fechaFin = value; }
        }

        private int _km;

        public int km
        {
            get { return _km; }
            set { _km = value; }
        }

        private string _tipoKm;

        public string tipoKm
        {
            get { return _tipoKm; }
            set { _tipoKm = value; }
        }

        private double _importe;

        public double importe
        {
            get { return _importe; }
            set { _importe = value; }
        }

        private string _tipoImporte;

        public string tipoImporte
        {
            get { return _tipoImporte; }
            set { _tipoImporte = value; }
        }

    }
}
